app/models/ems_event.rb

```diff
@@ -227,6 +227,7 @@ def get_target(target_type)
 
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
+    target_type = "middleware_server"   if event.event_type == "hawkular_event"
```

Comment:
Unless the logic have been changed recently the generic hawkular_events were not shown on the timeline, but anyway. We are agree that we will refine the events labels in a follow up task.

I am not sure that it will work nicely to use a single target type.
MiQ maps alerts with models, so hence we need to follow that, perhaps that could have some enhancement in the implementation. Perhaps in the event stream that can be mapped in some middleware_entity with a column for the type, but that need to be studied as there are several places where that could affect.